config/tag-processor-test.yaml

```
@@ -0,0 +1,45 @@
+    endpoint: "opc.tcp://localhost:50000"
+    nodeIDs: ["ns=6;s=DataAccess_AnalogType"]
+    subscribeEnabled: true
+    useHeartbeat: true
+    browseHierarchicalReferences: true
+
+pipeline:
+  processors:
+    - tag_processor:
+        defaults: |
+          msg.meta.location_path = "enterprise.plant1";
+          msg.meta.data_contract = "_historian"; 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)